mat-process-api.Tests/V1/Helper/MatProcessDataHelper.cs

```diff
@@ -17,17 +18,38 @@ public static MatProcessData CreateProcessDataObject()
 
             return new MatProcessData
             {
+                Id = faker.Random.Guid().ToString(),
+                ProcessType = new ProcessType()
+                {
+                    value = faker.Random.Int(),
+                    name = faker.Random.Word()
+                },
                 DateCreated = faker.Date.Recent(),
                 DateLastModified = faker.Date.Recent(),
-                ProcessData = {},
-                ProcessDataSchemaVersion = faker.Random.Int(0,10),
```

Comment:
Combined with "Validators/PostInitialProcessDocumentRequestValidatorTests.cs" Line 39. This has a probability of 10% of failing any controller test that uses this helper. Due to validation fire-ing off upon SchemaVersion = 0, when it's not intended (as judging by test's purpose). The starting Schema version will be '1', so need to exclude zero anyway.

Reply:
Changed it to generate numbers from 1 to 10 instead, not to hit zero.